assets/components/src/button-group/style.scss

```diff
@@ -23,12 +23,17 @@
 		&.is-pressed {
 			position: relative;
 			z-index: 2;
-			background-color: $primary-050 !important;
+			background-color: white !important;
```

Comment:
Same question as with ButtonCard—would there ever be a scenario where `white` is an inappropriate background color?

That said, I couldn't actually find any scenario in our code that uses an `is-pressed` class on a Newspack button component, so I wonder if we even need these styles?

Reply:
It's currently not being used but I've planned to use it during the onboarding. And again, I don't think the background-color should be an issue